### ISSUE TITLE
Luci Web Interface

### ISSUE THREAD

**Issue by Delinex49** [hwti/G-010S-A#43]
Hello, friends.
I have a question about Luci Web Interface. 
I'm watching videos of how people set up their G-010S-A, and everyone has a Luci Web Interface. 
Meanwhile, I have some weird GPON Home Gateway page.
I installed firmware 3FE46398BGCB22 (BGC.B22) and nothing has changed. 
Do you have any ideas on how I can get Luci Web Interface on my G-010S-A?
Thanks!

**Comment by lineardraft** [comment]
+1 I would be interested too.

**Comment by jspavlick** [comment]
+1 interested, too. It seems as though in this video, Luci is installed (but mostly in Chinese?) https://www.youtube.com/watch?v=3g6mRDfUppQ&t=0s

**Comment by MerEsc** [comment]
All firmwares with LuCi is a custom chinese firmware. I find some firmwares and upload to my cloud, https://cloud.asenheim.org/s/tqGTgBDSZgoKFyg

**Comment by jspavlick** [comment]
Thank you. These seem even seem to be newer versions than what I found on my AAAA the other day and posted here on GitHub as an issue. I will investigate and tinker with updating one of my SFP sticks (I have several at this point) with one of these firmwares and see where I get!

**Comment by aryanrajseo** [comment]
is there any stable version of openwart with LuCI in english with guide to flash it on G-010S-A? i don't see any guide but there are some videos with openwrt pre-installed.

**Comment by mikegleasonjr** [comment]
@MerEsc can you provide some commands to flash those?

**Comment by jspavlick** [comment]
Flashing is pretty easy. It is covered here I know at least: https://hack-gpon.org/ont-nokia-g-010s-a/

**Comment by mikegleasonjr** [comment]
So I flash the file `nokia-g010sa_new-squashfs.image` exactly as the firmwares in this repo? ok..

**Comment by jspavlick** [comment]
No, not if you want LUCI. If you want LUCI you need to use one of these builds. https://cloud.asenheim.org/s/tqGTgBDSZgoKFyg they're all slightly different and the README's are in chinese as I recall. This repo (https://github.com/hwti/G-010S-A/tree/main/firmwares) contains the "official" firmwares from Nokia. The LUCI builds are "unofficial"

**Comment by mikegleasonjr** [comment]
ok thanks, yes that's what I meant, the file `nokia-g010sa_new-squashfs.image` was from the https://cloud.asenheim.org/s/tqGTgBDSZgoKFyg repo!

Thanks again!

EDIT: it worked!

**Comment by aryanrajseo** [comment]
> ok thanks, yes that's what I meant, the file `nokia-g010sa_new-squashfs.image` was from the https://cloud.asenheim.org/s/tqGTgBDSZgoKFyg repo!
> 
> Thanks again!
> 
> EDIT: it worked!

hey, great, could you share the screencast or a guide for the same? Thanks

**Comment by aryanrajseo** [comment]
hi @jspavlick @mikegleasonjr , i want to flash openwrt on Nokia G-010S-A, could you help with the tools needed with a screencast or guide? thanks

**Comment by jspavlick** [comment]
Please visit the hack-gpon link I posted earlier. I believe this video shows how to use the LUCI GUI but not the flashing? I Haven't watched it in a while  https://www.youtube.com/watch?v=3g6mRDfUppQ

**Comment by aryanrajseo** [comment]
hi @jspavlick  the video is not have any guide to flash the openwrt. are you pointing out this guide https://github.com/hwti/G-010S-A?tab=readme-ov-file#upgrade ? we don't need UART or any other tool and can be done with ssh only?

===

Upgrade
To flash a different firmware, the first step is to determine the current running image (0 or 1) :

ONTUSER@SFP:~# upgradestatus
***** get current running image *****
current running image is  image0 !
...
Then the other image must be flashed, the bootloader configured to boot it, and we can reboot :

mtd write image.bin image1
update_env_flag 1
reboot

**Comment by jspavlick** [comment]
Ssh and scp as I recall. https://hack-gpon.org/ont-nokia-g-010s-a/

**Comment by Payti** [comment]
> All firmwares with LuCi is a custom chinese firmware. I find some firmwares and upload to my cloud, https://cloud.asenheim.org/s/tqGTgBDSZgoKFyg

@MerEsc have You maybe latest custom LuCi FW compilation: **基于新版固件修改版_2026.03.01.zip**  from -->> https://pan.baidu.com/s/1we_EvxYRimPEOOJyGKBt4w?pwd=s14g#list/path=%2FMA5671A%E3%80%81ALCATEL%E5%8F%8ANOKIA%E5%9B%BA%E4%BB%B6 ?

If yes please share on Your cloud ...

Thanks and regards,
Tom

**Comment by essarouri** [comment]
@Payti I've mirrored the folder contain latest version (基于新版固件修改版_2026.03.01.zip) from Baidu Pan to Mega for easier access.

Direct Links:

- Mega Mirror: https://mega.nz/folder/7nIklLhC#8P0ECIx3qz8cDLIO1ue3Ag

- Original Baidu Pan: https://pan.baidu.com/s/1we_EvxYRimPEOOJyGKBt4w (Pass: s14g)

- Original Article: https://www.right.com.cn/forum/thread-8220173-1-1.html

How to download from Baidu Pan (Free solution):
I spent a lot of time looking for a way to skip the paid services and account restriction. This tool actually worked for me:

- Tool: https://www.uy5.net/ode/

- Tutorial: https://cyber.suma.tw/baidu-pan-no-account-dl/

- Video Guide: https://youtu.be/eVq9TqGfCw8?si=gvMG6D44K34rvY22

NOTE: I didn't create this app, so please be safe—run it in a Virtual Machine (VM) or an isolated environment just in case!
